pkg/skaffold/build/kaniko/sources/localdir.go

```diff
@@ -24,7 +24,7 @@ import (
 	"path/filepath"
 
 	"github.com/pkg/errors"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Can someone explain me why `golangci-lint` wants this? Goland removed this since this is an unnecessary import alias. Running `goimports` didn't fix this and I had to add it back manually.

Reply:
So, did a bit of research on this: 
1.) goimports _should_ fix it - that's why `golangcli-lint` detects it
2.) goimports generates an alias to packages where the "real package name" defined in the go file on disk does not equal to the "basic package name" (https://github.com/golang/tools/blob/master/imports/fix.go#L402). Now the "basic package name" is  usually the last folder (base) in the import path, except that if the path ends in foo/vN, it assumes the package name is foo (see [importPathToBasic](https://github.com/golang/tools/blob/d00ac6d27372a4273825635281f2dc360d4be563/imports/fix.go#L672-L674)) 

Now - to keep things simple we rely on tools like goimports. 
Here it seems that the k8s code is deviating from the assumed format of goimports - i.e. the package name is not `core` under the `v1` folder...which creates the above issue.

Reply:
This is fixed by very recent versions of goimports. You might want to update with `go get -u golang.org/x/tools/cmd/goimports`

Reply:
For some reason running `go get golang.org/x/tools/cmd/goimports` without the `-u` flag doesn't install the latest version if its not installed yet. But I think it would be nice to have a `Makefile` target which goimports/gofmt's all files (including checking for the latest version of `goimports` 😉). 

@balopat Thanks for the explanation 👍